frontend/src/reducer/publicationList.js

```diff
@@ -24,51 +25,84 @@ function uniqByContentId (array) {
   return uniqBy(array, 'id')
 }
 
-const defaultPublicationList = []
+const defaultPublicationPage = {
+  list: [],
+  nextPageToken: '',
+  hasNextPage: false
+}
 
-export default function publicationList (state = defaultPublicationList, action) {
+export default function publicationList (state = defaultPublicationPage, action) {
```

Comment:
Suggestion : publicationPage instead of publicationList
You can just resolve if you don't want to apply.

Reply:
I hesitated as this state is not only one page, but this naming is still better than `List`, I'll rename this.

Reply:
Renamed.